aria_plugin/operations.py

```diff
@@ -30,6 +31,14 @@
 @operation
 def create(**_):
     env = Environment(ctx)
+    # Make sure no other service was created under the same name
+    if env.model_storage.service_template.list(
```

Comment:
explain why we don't just except the exception that store service template raises when we have a blueprint with the same name stored.